src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx

```diff
@@ -31,21 +34,35 @@ export const ViewEvaluationHeader = () => {
 
   return (
     <>
-      <div className='text-right'>
-        <h1>Evaluation Schedule</h1>(
-        {formatDate(evaluation_administration?.eval_schedule_start_date)} to{" "}
-        {formatDate(evaluation_administration?.eval_schedule_end_date)})
-      </div>
       <div className='flex flex-col'>
-        <div className='flex justify-between items-center  mt-2'>
+        <div className='flex flex-col justify-between items-start md:items-end mt-2 md:flex-row gap-4'>
           <div>
-            <h1 className='text-2xl font-bold'>
-              {evaluation_administration?.name}
-            </h1>
-            <div>
-              Evaluation Period (
-              {formatDate(evaluation_administration?.eval_period_start_date)} to{" "}
-              {formatDate(evaluation_administration?.eval_period_start_date)})
+            <div className='flex gap-4 primary-outline items-end mb-4'>
+              <PageTitle>{evaluation_administration?.name}</PageTitle>
+              <Badge
+                size='medium'
```

Comment:
remove this after you set the default size in badge

Reply:
but I think you need to update other components that are using `small` badge

Reply:
Ahh so set the default to `medium`, then add a size variant for `small `badges?